### ISSUE TITLE
fail on selfdestruct

### ISSUE THREAD

**Issue by snd** [crytic/echidna#26]
i tried fuzzing a contract that anyone can "kill".
echidna didn't fail.
not sure if this is by design. i think it would make sense to have echidna fail if it manages to  `selfdestruct` a contract. (maybe make this the default but add an option to allow it.)

contract `self_destruct.sol`:

```solidity
contract SelfDestruct {
  function kill(address _to) public {
      selfdestruct(_to);
  }
  function echidna_alwaystrue() public pure returns (bool) {
      return true;
  }
}
```

run:

```
$ echidna-test self_destruct.sol
self_destruct.sol:1:1: Warning: Source file does not specify required compiler version!Consider adding "pragma solidity ^0.4.21;"
contract SelfDestruct {
^ (Relevant source part starts here and spans across multiple lines).
━━━ self_destruct.sol ━━━
  ✓ "echidna_alwaystrue" passed 10000 tests.
  ✓ 1 succeeded.
```

commit: 83b8fd5e8f57c825c98370efa802922da412dada

**Comment by japesinator** [comment]
hmmm, it definitely shouldn't do that! I'll take a look this week

**Comment by ggrieco-tob** [comment]
Hi @snd. Sorry for the long delay to address this issue! Can you please test PR #131 ?

**Comment by ggrieco-tob** [comment]
Fixed in #515
